slang.py

```diff
@@ -16,7 +16,7 @@ def de_emojify(self, s):
         return ''.join(filter(lambda x: x in printable, str(s)))
 
     @commands.command(name='slang', brief='Gives the meaning of the slang from UrbanDictionary', case_insensitive=True)
-    @commands.cooldown(1, 60, commands.BucketType.user)
+    @commands.cooldown(1, 40, commands.BucketType.channel)

```

Comment:
regression: now one user can lead to cooldown for others.

Reply:
Yes, it's pick your poison. If you want the channel to be spam free, then this trade-off is inevitable.